app.json

```diff
@@ -59,6 +59,18 @@
     },
     "RAILS_ENV": {
       "required": false
+    },
+    "OSEM_TWITTER_CONSUMER_KEY": {
+      "required": true
```

Comment:
app.json file is only used for heroku review apps, is it not? 
That in practice means that before creating a new review app, we need to set those keys. It worked a couple of times, last time I tried it, it did not :/
I am not sure it makes much sense to have this for review apps though - unless of course we figure out the right values to put there.